app/gui2/src/util/shortcuts.ts

```diff
@@ -391,6 +391,9 @@ export function defineKeybinds<
     >,
   ): (event: Event_, stopAndPrevent?: boolean) => boolean {
     return (event, stopAndPrevent = true) => {
+      // Do not handle repeated keyboard events (held down key).
+      if (event instanceof KeyboardEvent && event.repeat) return false
```

Comment:
I'm not sure what else we could do here. Returning `true` is not really correct either, as we don't handle the event. My reasoning was that since all handlers should perform this check, they all should not do anything else. This is currently true, since we always use the shortcut definitions when working with keyboard events. On the other hand, it leaves the possibility to explicitly allowing repeat key handling when we need it.